third_party/intel/backend/driver.py

```diff
@@ -28,21 +27,27 @@ def find_sycl(include_dir: list[str]) -> tuple[list[str], Optional[str]]:
       AssertionError: if library was not found.
     """
     include_dir = include_dir.copy()
+    sycl_dir = None
     assertion_message = ("sycl headers not found, please install `icpx` compiler, "
                          "or provide `ONEAPI_ROOT` environment "
                          "or install `intel-sycl-rt>=2025.0.0` wheel")
-
-    if shutil.which("icpx") and os.name != "nt":
+    icpx_path = shutil.which("icpx")
+    if icpx_path and os.name != "nt":
         # only `icpx` compiler knows where sycl runtime binaries and header files are
-        return include_dir, None
+        icpx_dir = os.path.dirname(icpx_path)
+        compiler_root = os.path.dirname(icpx_dir)
+        include_dir += [os.path.join(compiler_root, "include"), os.path.join(compiler_root, "include/sycl")]
```

Comment:
Why do we need compiler top level include?
```suggestion
        include_dir += [os.path.join(compiler_root, "include/sycl")]
```

Reply:
Similar to 
```python
    if oneapi_root:
        include_dir += [
            os.path.join(oneapi_root, "compiler/latest/include"),
            os.path.join(oneapi_root, "compiler/latest/include/sycl")
        ]
```